src/index.html

```diff
@@ -1009,6 +1009,23 @@ <h2 class="app__title">
 			</style>
 		</a>
 
+		<a class="list__item app js-app"
+				href="https://resilientwebdesign.com/"
+				data-app="ResilientWebDesign"
```

Comment:
We usually keep all app ids and icon file names the same: lowercase, separated with dashes:

- paper-planes
- smaller-pictures
- puzzle-2048

Would you mind doing the same? I mean: `data-app="resilient-web-design"` and `resilient-web-design.svg`.